### DISCUSSION TITLE
Now playing auto message

### DISCUSSION THREAD

**Discussion by alerey1610** [eritislami/evobot#1111] [General]
It would be nice if there's an command or a option in the config for disabling the now playing message.
Like if i play a message i always get a discord message cuz of the announcment of the new song and the chat is getting spammed with it.

**Comment by Velosofy** [comment]
The previous song message is deleted, then the bot send the new embedded message with the new song

**Comment by eritislami** [comment]
Enable pruning via the pruning command or from the config.json, you will still get pinged but the message will be deleted after song ends.

`"PRUNING": true,`
